test/util/fake-xml-http-request-test.js

```diff
@@ -1063,18 +1063,39 @@ if (typeof window !== "undefined") {
                 assert.equals(this.xhr.responseHeaders, {});
             });
 
-            it("sets state to DONE if sent before", function () {
-                var readyState;
+            it("sets state to DONE if aborted in-flight", function () {
+                var statechanges = [];
                 this.xhr.open("GET", "/");
+                this.xhr.onloadstart = function () {
+                    this.abort();
+                };
+                this.xhr.onreadystatechange = function () {
+                    statechanges.push(this.readyState);
+                };
+
                 this.xhr.send();
 
-                this.xhr.onreadystatechange = function () {
-                    readyState = this.readyState;
+                assert.equals(statechanges, [FakeXMLHttpRequest.OPENED, FakeXMLHttpRequest.DONE]);
+            });
+
+            it("triggers loadend if aborted in-flight", function () {
+                this.xhr.open("GET", "/");
+                this.xhr.onloadend = sinonStub();
+                this.xhr.onloadstart = function () {
+                    this.abort();
                 };
+                this.xhr.send();
+
+                assert.isTrue(this.xhr.onloadend.called);
+            });
+
+            it("sets state to DONE if sent before", function () {
```

Comment:
I don't see how the test description matches the content? You are not referring to DONE anywhere.